### PR TITLE
Create unit tests for pilot class

### PR DESCRIPTION
**Pull request by kivipe**
DCS Liberation does not have much unit tests. Pilot class is not very exiting but it was a good place to start.

### CODE REVIEW

File: tests/test_pilot.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+def test_missions_flown(pilot) -> None:
+    pilot.record.missions_flown = 1
+    assert pilot.record.missions_flown == 1
+
+
+def test_random_pilot_name() -> None:
+    faker = Faker()
+    random_pilot = Pilot.random(faker)
+    assert len(random_pilot.name) > 3
```
Comment (DanAlbert) [inline tests/test_pilot.py]: As someone with a three letter name, I don't like this particular test ;)

It's not really safe to assume anything about a randomly generated name (https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/ is a fun read). `assert random_pilot.name` is good enough.

```suggestion
    assert random_pilot.name
```

**Review by DanAlbert — APPROVED** [review]
People like you are my heroes. I don't even care that this particular bit of coverage is low hanging fruit, this is my favorite kind of PR <3

**Comment by kivipe** [comment]
Mypy failed. I made a fix that passed locally and pushed it in.

File: tests/test_pilot.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+def test_missions_flown(pilot: Pilot) -> None:
+    pilot.record.missions_flown = 1
+    assert pilot.record.missions_flown == 1
+
+
+def test_random_pilot_name() -> None:
+    faker = Faker()
+    random_pilot = Pilot.random(faker)
+    assert len(random_pilot.name) > 3
```
Comment (DanAlbert) [inline tests/test_pilot.py]:
```suggestion
    assert random_pilot.name
```

**Pull request merged**